GettingStarted.md

```diff
@@ -0,0 +1,152 @@
+# Introduction
+Welcome to impress.js! This presentation framework allows you to create stunning presentations with the power of CSS3 transformations. It is inspired by the idea behind [prezi.com](https://prezi.com). 
+**NOTE:** This Guide is not made for you, if you have never written HTML and/or CSS before. Knowing your way around in JavaScript certainly helps, but is not a necessity. You may still continue this tutorial and try to understand what we do as you go. 
+
+Also, if you are an experienced (or sorta experienced user, anyway), you might prefer the [DOCUMENTATION](DOCUMENTATION.md).
+
+# Getting started with impress.js
+## Installation / acquiring the framework
+You first need to decide if you want to use the bleeding edge (most up to date, generally very reliable as well, but may contain more problems) or the stable release. You can use one of the methods mentioned below or the direct download link to only the javascript file. The benefit is, you don't have to find the file in the repository and you can copy the link and directly load the file from the cdn. 
+
+### **Direct links to only impress.js file**
+- V2.0.0: https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js
+- V1.1.0: https://cdn.jsdelivr.net/gh/impress/impress.js@1.1.0/js/impress.js
+- Source: https://cdn.jsdelivr.net/gh/impress/impress.js/js/impress.js
+
+For older versions, please just replace the version number behind the @!
+
+**Loading the script directly from cdn**
+Loading the script from the cdn is quite straight forward. You need to include a script tag in the head of the HTML document:
+
+```
+<script src="https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js"></script>
+```
+
+### **Stable release:**
+Head to the releases tab and download the source code as zip or as a tarball. Go ahead and unzip / untar it. You may find the framework under ```/js/impress.js.```
+
+### **Bleeding Edge:**
+In the github repository, click the "Code"-Button, then click "Download ZIP". Go ahead and unzip / untar it. You may find the framework under ```/js/impress.js.```
+
+## Setting up the project
+Open up your favorite text-editor / IDE, for example Visual Studio Code, Atom, Notepad ++, ...
+Now, copy the *impress.js* file into the folder you are working in and create a new file called *index.html* and create the basic HTML layout:
+
+```
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+</head>
+<body class="impress-not-supported">
+    <div class="fallback-message">
+        <p>Your browser <b>doesn't support the features required</b> by impress.js, so you are presented with a simplified version of this presentation.</p>
+        <p>For the best experience please use the latest <b>Chrome</b>, <b>Safari</b> or <b>Firefox</b> browser.</p>
+    </div>
+
+    <div id="impress">
+        <div id="myFirstSlide" class="step">
+            <h1>My first Slide</h1>
+        </div>
+    </div>
+
+    <script type="text/javascript" src='./impress.js></script>
```

Comment:
Shouldn't you also use the online file (jsdelivr) here too?

Reply:
Technically yes. I thought, I could show both versions.

Reply:
I think it's better to be consistent.